### DISCUSSION TITLE
关于绳网作为二创基地的可能性

### DISCUSSION THREAD

**Discussion by UranusV** [share121/inter-knot#2162] [其他]
![PaimonMeme1](https://github.com/user-attachments/assets/e7f823f9-25f5-40f5-991a-7badf49a5836)
感觉可以有！实际上也已经看到有在发布作品的小伙伴了。但是如果向二创方向发展就需要一些特定的算法推送，而现在这种不受数据干涉的浏览状态也十分难得...不知道大家怎么看？

**Comment by HifumiMizuhara** [comment]
我认为好。B站现在受浏览量影响太大了，小UP完全没流量

**Reply by UranusV** [reply]
的确是这样 现在的b站过于商业化了 对二创来说反而差点意思

**Comment by March3th** [comment]
等优化好搜索、分区、收藏等基础功能之后可以试一试

**Comment by MissHedwig** [comment]
我认为好，其实不错的，来这边需要点操作的（

**Reply by UranusV** [reply]
确实 有门槛这一点对于清净的社区氛围而言反而是优势

**Comment by Yuesekaer** [comment]
够呛，Github一般正常情况能否正常访问都要打一个问号

**Reply by HifumiMizuhara** [reply]
正是这样可以保证社区氛围，成本太低就容易变成今天你米游社那样

**Comment by LHk416** [comment]
我觉得可以

**Comment by wugan-bot156** [comment]
不太行，这里的人实在是太少了，绳网顶多能被一些新人作者用来宣传一下自己。

**Comment by 00zzxx** [comment]
我觉得可以，毕竟有点门栏可以保证安全性，不至于被贴脸骂原批什么的...但是有门栏也必然会筛掉大部分懒人，访问也不能一直保持稳定

**Comment by Kioeco** [comment]
看后续更新吧，有好意见的大佬，估计能整出不一样的花火

**Comment by ArcSurge** [comment]
如果不考虑流量可以考虑，现在注册都有一定门槛

**Comment by HKLHaoBin** [comment]
估计很不好做，因为评论是基于 github 的，所以很多规则都要遵循 github 的规则，这能给开发者极大的便利，因为不需要考虑安全性，但是可开发性很小，综上所述，二创的话还得是B站。

<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Eurus1997** [comment]
可以有，但是视频文件可能有点问题<img src="https://upload-bbs.mihoyo.com/upload/2022/11/24/78c0c40f7cc89facd10ff38b3385d918_3614720584318019134.png" alt="纳西妲-沉思中" width="75" >
